src/search_algo.hpp

```diff
@@ -533,19 +563,31 @@ search(LocalDataHolder<TGlobalHolder> & lH)
 #endif
                 for (auto [ subjNo, subjOffset ] : cursor.lazy_locate())
                 {
+                    bool const fwd = (subjNo % 2) - 1;
                     TMatch m {static_cast<typename TMatch::TQId>(i),
                               static_cast<typename TMatch::TSId>(subjNo),
                               static_cast<typename TMatch::TPos>(seedBegin),
                               static_cast<typename TMatch::TPos>(seedBegin + seedLength),
                               static_cast<typename TMatch::TPos>(subjOffset),
-                              static_cast<typename TMatch::TPos>(subjOffset + seedLength)};
+                              static_cast<typename TMatch::TPos>(subjOffset + seedLength),
+                              fwd};
```

Comment:
Why do we still have these changes? Can't we compute `fwd` from `subjNo` (like we do above) wherever `fwd` is used?